lib/digest.js

```diff
@@ -0,0 +1,63 @@
+/*!
+ * Copyright (c) 2022 Digital Bazaar, Inc. All rights reserved.
+ */
+import jsonld from 'jsonld';
+import {encode as base58Encode} from 'base58-universal';
+import {sha256digest} from './sha256digest.js';
+
+const URDCA_2015_ALG = 'urdca2015';
+// Multicodec table entry code 0xb403, converted to varint
+const URDCA2015_HEADER = new Uint8Array([131, 232, 2]);
```

Comment:
I'm just presuming this is right. :)

Reply:
@dlongley - hehe, I got it from https://github.com/multiformats/js-multiformats